src/index.html

```diff
@@ -6,8 +6,39 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./images/imac.jpeg">
+    <link rel="stylesheet" href="./styles/stars.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <img src="./images/imac.jpeg" alt="logo"
+        class="image-container">
```

Comment:
```suggestion
      <img 
        src="./images/imac.jpeg" 
        alt="logo"
        class="image-container"
      >
```
Consider fixing indents according to the code style guide